src/routes/api/auth.js

```diff
@@ -0,0 +1,33 @@
+const KoaRouter = require('koa-router');
+const jwtgenerator = require('jsonwebtoken');
+require('dotenv').config();
+
+const router = new KoaRouter();
+
+async function generateToken(user) {
+  return new Promise((resolve, reject) => {
+    jwtgenerator.sign(
+      { userId: user.id },
```

Comment:
Según el [estándar JWT](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.2), el claim correspondiente al "sujeto" al que hace referencia el token está dado por la propiedad `sub`. Creo que debería utilizarse eso aquí:
```suggestion
      { sub: user.id },
```

Reply:
Cambiado, gracias por la corrección!